docs/concepts/architecture/master-node-communication.md

```diff
@@ -17,6 +17,8 @@ customize their installation to harden the network configuration such that
 the cluster can be run on an untrusted network (or on fully public IPs on a
 cloud provider).
 
+![](/images/docs/communication.png)
```

Comment:
there are a lot of different things that communicate - would you consider renaming the files:
- communication.png
- communication.gv

to:
- master-node-communication.gv 
- master-node-communication.png

to help disambiguate these files from others in this directory?